packages/custom-functions-metadata/package.json

```diff
@@ -41,6 +40,7 @@
     "mocha": "^9.1.1",
     "office-addin-lint": "^2.0.0",
     "office-addin-prettier-config": "^1.1.4",
+    "office-addin-usage-data": "^1.4.6",
```

Comment:
Is this not a regular dependency anymore?

Reply:
What do you mean? It appears as a dependency here.
Or do you mean devDependency vs other types?

Reply:
cli was a dependency and you're replacing it with usage-data as a dev dependency.  If it was just getting the same method from a different package, I would think the nature of the dependency of it wouldn't change.

Reply:
My mistake, making it a regular dependency again